pkg/common/configs/configvalidator.go

```diff
@@ -377,7 +377,7 @@ func Validate(newConfig *SchedulerConfig) error {
 			partition.Name = DefaultPartition
 		}
 		// check the queue structure
-		log.Logger().Debug("checking partition",
+		log.Logger().Info("checking partition",
```

Comment:
Do we need to include this in the INFO level, this line of log is not very informative won't be very much useful.

Reply:
Remove the unnecessary log.